tests/ops/integration_tests/saas/test_jira_task.py

```diff
@@ -0,0 +1,159 @@
+    graph = DatasetGraph(merged_graph)
+
+    v = await graph_task.run_access_request(
+        privacy_request,
+        policy,
+        graph,
+        [jira_connection_config],
+        {"email": jira_identity_email},
+        db,
+    )
+
+    assert_rows_match(
+        v[f"{dataset_name}:customers"],
+        min_size=1,
+        keys=[
+            "self",
+            "accountId",
+            "accountType",
+            "emailAddress",
+            "avatarUrls",
+            "displayName",
+            "active",
+            "locale",
+        ],
+    )
+
+    # verify we only returned data for our identity email
+    # assert v[f"{dataset_name}:customers"][0]["displayName"] == jira_user_name
+    user_id = v[f"{dataset_name}:customers"][0]["accountId"]
+
+
+@pytest.mark.integration_saas
+@pytest.mark.integration_jira
+@pytest.mark.asyncio
+async def test_jira_erasure_request_task(
+    db,
+    policy,
+    erasure_policy_string_rewrite,
+    jira_connection_config,
+    jira_dataset_config,
+    jira_erasure_identity_email,
+    jira_create_erasure_data,
+) -> None:
+    """Full erasure request based on the jira SaaS config"""
+
+    masking_strict = CONFIG.execution.masking_strict
+    CONFIG.execution.masking_strict = False  # Allow Delete
+
+    privacy_request = PrivacyRequest(
+        id=f"test_jira_erasure_request_task_{random.randint(0, 1000)}"
+    )
+    identity = Identity(**{"email": jira_erasure_identity_email})
+    privacy_request.cache_identity(identity)
+
+    dataset_name = jira_connection_config.get_saas_config().fides_key
+    merged_graph = jira_dataset_config.get_graph()
+    graph = DatasetGraph(merged_graph)
+    
+    v = await graph_task.run_access_request(
+        privacy_request,
+        policy,
+        graph,
+        [jira_connection_config],
+        {"email": jira_erasure_identity_email},
+        db,
+    )
+
+    assert_rows_match(
+        v[f"{dataset_name}:customers"],
+        min_size=1,
+        keys=[
+            "self",
+            "accountId",
+            "accountType",
+            "emailAddress",
+            "avatarUrls",
+            "displayName",
+            "active",
+            "locale",
+        ],
+    )
+
+    x = await graph_task.run_erasure(
+        privacy_request,
+        erasure_policy_string_rewrite,
+        graph,
+        [jira_connection_config],
+        {"email": jira_erasure_identity_email},
+        get_cached_data_for_erasures(privacy_request.id),
+        db,
+    )
+
+    assert x == {
+        f"{dataset_name}:customers": 1,
+    }
+
+    jira_secrets = jira_connection_config.secrets
+    base_url = f"https://{jira_secrets['domain']}"
+
+    sleep(60)
+
+    # user
+    response = requests.get(
+        url=f"{base_url}/rest/api/3/user/search",
+        params={"query": jira_erasure_identity_email},
+        auth=HTTPBasicAuth(
+            f"{jira_secrets['username']}", f"{jira_secrets['api_key']}"
+        ),
```

Comment:
This can be written as just a tuple `auth=(jira_secrets['username'], jira_secrets['api_key'])`